src/components/NavButton/index.tsx

```diff
@@ -13,32 +12,39 @@ export interface NavButtonProps extends ButtonProps {
     isDisabled?: boolean;
 }
 
-const NavButton: React.FC<NavButtonProps> = ({
-    className,
-    titleText,
-    buttonType = ButtonClass.Action,
-    icon,
-    clickHandler,
-    isDisabled,
-    ...props
-}) => {
-    // NavButtons default to action button styling, provide secondary or primary to override
-    const buttonClassNames = classNames(
-        className,
-        styles.navButton,
-        styles[buttonType],
-        { [styles.disabled]: isDisabled }
-    );
+const NavButton = forwardRef<HTMLButtonElement, NavButtonProps>(
```

Comment:
The only real change is this, 26, and 38 where we are forwarding the ref to the <button>